components/org.wso2.carbon.security.caas/src/main/java/org/wso2/carbon/security/caas/user/core/bean/Attribute.java

```diff
@@ -21,18 +21,9 @@
  */
 public class Attribute {
 
-    private String identityStoreId;
     private String attributeName;
     private String attributeValue;
 
-    public String getIdentityStoreId() {
-        return identityStoreId;
-    }
-
-    public void setIdentityStoreId(String identityStoreId) {
-        this.identityStoreId = identityStoreId;
-    }
-
```

Comment:
We may needs to keep this. Otherwise how can we find where this attribute originates ?